DESCRIPTION

```diff
@@ -20,16 +20,26 @@ Depends:
     dplyr
 Imports:
     methods,
+    ggplot2,
     lazyeval,
     purrr,
     tibble,
+    rlang,
     stringr,
-    rlang (>= 0.1.2)
+    viridisLite
 Suggests:
     covr,
-    httptest,
+    httptest (>= 3.0.0),
     magrittr,
-    spelling
+    spelling,
+    vdiffr,
+    knitr,
+    testthat,
+    rmarkdown
+Remotes:
```

Comment:
Neither of these Remotes are necessary anymore, are they?

Reply:
They shouldn't be